src/administration/integrations/blackfire.md

```diff
@@ -40,9 +40,6 @@ Paste the credentials above in your `.platform.app.yaml` as follows:
 runtime:
     extensions:
         - name: 'blackfire'
-          configuration:
-              server_id: '<insert your Server ID>'
-              server_token: '<Insert your Server Token'
```

Comment:
Why the switch away from inline definition to env vars?  I'm sure both work, but why remove this version?

Reply:
This is a rather old/not very flexible/powerful way to do it compared to env vars.

Best is to promote the most up to date way. And to promote only one way, in order to avoid confusion ;)